Samples/StreamRecorder/StreamRecorderApp/RMCameraReader.cpp

```diff
@@ -102,6 +105,45 @@ void RMCameraReader::CameraUpdateThread(RMCameraReader* pCameraReader, HANDLE ca
     }
 }
 
+void SaveAccel(IResearchModeSensorFrame* pSensorFrame, IResearchModeAccelFrame* pSensorAccelFrame)
+{
+    DirectX::XMFLOAT3 sample;
+    char printString[1000];
+    HRESULT hr = S_OK;
+
+    ResearchModeSensorTimestamp timeStamp;
+    UINT64 lastSocTickDelta = 0;
+    UINT64 glastSocTick = 1;
+
+    pSensorFrame->GetTimeStamp(&timeStamp);
+
+    if (glastSocTick != 0)
+    {
+        lastSocTickDelta = timeStamp.HostTicks - glastSocTick;
+    }
+    glastSocTick = timeStamp.HostTicks;
+
+    hr = pSensorFrame->QueryInterface(IID_PPV_ARGS(&pSensorAccelFrame));
+    if (SUCCEEDED(hr))
+    {
+        hr = pSensorAccelFrame->GetCalibratedAccelaration(&sample);
+        if (FAILED(hr))
+        {
+            return;
+        }
+        sprintf(printString, "####Accel: % 3.4f % 3.4f % 3.4f %f %I64d %I64d\n",
+            sample.x,
+            sample.y,
+            sample.z,
+            sqrt(sample.x * sample.x + sample.y * sample.y + sample.z * sample.z),
+            (lastSocTickDelta * 1000) / timeStamp.HostTicksPerSecond
+        );
+        //OutputDebugStringA(printString);
```

Comment:
Anstelle eines (auskommentierten) Debug Outputs könntest du deine Ergebnisse in eine Ausgabedatei schreiben.
Vergleiche hierzu bspw. die SaveVLC() Methode

Reply:
so: `m_tarball->Addfile(printString);`?

Reply:
Also meine Überlegung dazu war: Ich muss das ja jetzt nicht "Convert the software bitmap to raw bytes", da ich das ja als String schon habe und die .tar Datei wird ja erfolgreich erstellt, also brauch ich den output file nicht einen Namen geben... dann müsste doch der gennante Code in die Datei schreiben?

Reply:
Eine Tar Datei ist ja nur ein Paket von mehreren anderen Dateien. Wenn du also dein Text in eine Datei (in dem Tar-Paket) schreibst musst du der Datei also einen Namen geben.
Zudem hat AddFile ein festes Interface (Siehe Tar.h).
Vermutlich musst du also dennoch als erstes einen Dateinamen angeben, als zweites den String und als drittes die Länge des Strings

Reply:
`std::string IMUData;
size_t outBufferCount = 0;
IMUData.insert(IMUData.end(), printString, printString);
        
m_tarball->AddFile(outputPath, &IMUData[0], IMUData.size());`

Ja, das versuche ich hier, aber dann heißt es "argument of type "char *" is incompatible with parameter of type "const uint8_t"

Reply:
Was wird denn als Fehler des aktuellen Stands ausgegeben?

Reply:
Severity	Code	Description	Project	File	Line	Suppression State
Error (active)	E0167	argument of type "char *" is incompatible with parameter of type "const uint8_t *"	StreamRecorder	C:\Users\Iori\Documents\GitHub\HoloLens2ForCV\Samples\StreamRecorder\StreamRecorderApp\RMCameraReader.cpp	393

Reply:
wupps sorry für die schlechte formatierung ^^